promscale/installation/kubernetes/helm.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Install Promscale on Kubernetes using Helm
+excerpt: Install Promscale on a Kubernetes cluster using Helm
+product: promscale
+keywords: [Kubernetes, analytics, Helm]
+tags: [install]
+related_pages: 
+  - /promscale/:currentVersion:/installation/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+
+# Install Promscale with Helm charts
+You can install Promscale using Helm charts and the Helm charts must be installed
+in this order:
+1. Install the TimescaleDB Helm chart
+1. Install the Promscale Helm chart
+
+## Before you begin
+1.  Understand the [resource requirements][resource-requirements] for your cluster.
+1.  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
```

Comment:
I do not think we should ask users to deploy K8s cluster. The header of this section says Kubernetes and this is part of the Kubernetes drop-down. Specifying Kubernetes explicitly sounds redundant and feels like Promscale needs its own K8s cluster.